sdk/monitor/monitor-query/src/generated/models/index.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import * as coreHttp from "@azure/core-http";
```

Comment:
@sarangan12 this is a bad generation, any idea what is going on here?

Reply:
@deyaaeldeen What is bad generation? This package is based on core V1.

Reply:
I ran `rushx generate:client`